Dockerfile.assisted-installer

```diff
@@ -11,7 +11,7 @@ RUN go mod download
 COPY . .
 RUN make installer
 
-FROM quay.io/centos/centos:centos8
+FROM registry.access.redhat.com/ubi8/ubi-minimal
```

Comment:
A question
Should we stick with ``latest`` tag? Or rather with something more stable (in the sense of non-changing) e.g. ``8.4``

Reply:
whoops, I actually meant to use 8.4, rather than latest. This reminds me that we should do the same in the image service too.